REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.2
```

Comment:
Since there's no version of Colors that currently supports 0.2, this should probably be changed to 0.3, yeah?